ebu_tt_live/bindings/converters/ebutt3_ebuttd.py

```diff
@@ -223,7 +329,15 @@ def convert_children(self, element, dataset):
                 output.append(copy.deepcopy(item.value))
             elif isinstance(item, ElementContent):
                 conv_elem = self.convert_element(item.value, dataset)
+
                 if conv_elem is not None:
+                    if isinstance(item.value, StyledElementMixin) and not isinstance(item.value, style_type):
+                        self._fix_fontsize(
```

Comment:
Right, so fixing the font size is its purpose, but what do we expect it to _do_? Does it normalise the font size to a canonical relative unit, or a canonical absolute unit, drop the horizontal component, etc.? 

This is a readability question not a functionality one - looking at this I see that if it is a StyledElementMixedIn and not a style element itself then we are going to fix the font somehow, but there's nothing telling me why I need to do this. Maybe a comment to say "font sizes in EBU-TT-D are single value percentages, which are always relative to the parent font's computed size, but the source data may have absolute and/or two value font sizes so convert whatever font size we have into a single % value in the context of its parent." ... or something less long-winded.